boot/initramfs_test.go

```diff
@@ -485,7 +485,7 @@ func (s *initramfsSuite) TestInitramfsRunModeChooseSnapsToMount(c *C) {
 		for _, t := range tt {
 			var cleanups []func()
 
-			comment := fmt.Sprintf("[%s] %s", tbl.name, t.comment)
+			comment := Commentf(fmt.Sprintf("[%s] %s", tbl.name, t.comment))
```

Comment:
Can't we format using Commentf directly instead of using fmt.Sprintf?

Reply:
ah yes good point, I just blindly applied this, I will fix that up